docs/contributing/community-contributions.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Community Contributions
+---
+
+A big part of what makes Gatsby great is each and every one of you in the community. Your contributions enrich the Gatsby experience and make it better every day.
+
+We welcome all contributions from you in the community, and would be thrilled to amplify your voice. Contributions are not limited to code, and can take all shapes and forms:
+
+- Your wonderful Gatsby.js website!
+- Gatsby starters
+- Gatsby plugins
+- Blog posts and case studies
+- Talk and lecture videos
+- Workshop materials
+- Github issues and pull requests
```

Comment:
Maybe link to the "good first issue" label for issues? (Not sold on linking to it exactly here, but it's probably worth linking to)

```suggestion
- Github issues and pull requests
    - Check out our [good first issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22)
```